resources/metadata/helmExecute.yaml

```diff
@@ -294,34 +245,43 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-      - name: chartRepo
-        type: string
-        description: set the chart repository
-        default: "https://charts.helm.sh/stable"
+      - name: customTlsCertificateLinks
+        type: "[]string"
+        description: "List of download links to custom TLS certificates. This is required to ensure trusted connections to instances with repositories (like nexus) when publish flag is set to true."
         scope:
           - GENERAL
           - PARAMETERS
           - STAGES
           - STEPS
-      - name: helmRegistryUser
-        type: string
-        description: set the user for login to helm registry
+      - name: lintFlag
+        type: bool
+        description: flag is used to enable helm lint command if value of helmCommand is empty.
+        default: false
         scope:
           - GENERAL
           - PARAMETERS
           - STAGES
           - STEPS
-      - name: helmChartServer
-        type: string
-        description: set chart server for pushing chart
-        default: "localhost:5000"
+      - name: packageFlag
+        type: bool
+        description: flag is used to enable helm package command if value of helmCommand is empty.
+        default: false
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: publishFlag
+        type: bool
+        description: flag is used to enable helm publish command if value of helmCommand is empty.
+        default: false
         scope:
           - GENERAL
           - PARAMETERS
           - STAGES
           - STEPS
   containers:
```

Comment:
add `artifactVersion` parameter to allow updating the version in the chart.yml

Reply:
not needed since update is done via step `artifactPrepareVersion`
https://github.com/SAP/jenkins-library/pull/3627